theme/static/css/elements.css

```diff
@@ -704,6 +704,17 @@ a.thumbnail:focus img {
 	margin-bottom: 50px;
 	padding: 15px 0;
 }
+
+.page-tip-error {
```

Comment:
All django.contrib.message types were being handled the same before this.  I added a new class for messages with type error to have red styling.  I pulled the color values from the styling for error messages on the resource landing page.